SteamcLog/Classes/SteamcLog.swift

```diff
@@ -13,46 +13,59 @@ import XCGLogger
 public struct SteamcLog {
     public var config: Config! {
         didSet {
-            sentryDestination.outputLevel = config.logLevel.remote.xcgLevel
+            sentryDestination?.outputLevel = config.logLevel.remote.xcgLevel
             fileDestination.outputLevel = config.logLevel.disk.xcgLevel
             systemDestination.outputLevel = config.logLevel.console.xcgLevel
         }
     }
 
+    // Sentry configuration options. If this is not set (or set to nil during creation), Sentry will not be used.
+    public var sentryConfig: SentryConfig?
+
     @usableFromInline internal var xcgLogger: XCGLogger!
     @usableFromInline internal let encoder = JSONEncoder()
 
     private var fileDestination: FileDestination!
-    private var sentryDestination: SentryDestination!
+    private var sentryDestination: SentryDestination?
     private var systemDestination: SteamcLogSystemLogDestination!
 
-    public init(_ config: Config) {
+    /*
+     * Create your SteamcLog object.
+     * We recommend setting up a global instance of SteamcLog, probably in your AppDelegate file.
+     *
+     * - Parameters:
+     *   - config: General configuration options. See Config.swift for details.
+     *   - sentryConfig: Sentry configuration options. If this is `nil`, Sentry will not be used.
+     */
+    public init(_ config: Config, sentryConfig: SentryConfig?) {
         self.config = config
         xcgLogger = XCGLogger(identifier: config.identifier, includeDefaultDestinations: false)
 
         xcgLogger.setup(
             level: config.logLevel.global.xcgLevel
         )
 
-        SentrySDK.configureScope { scope in
-            #if DEBUG
-            scope.setTag(value: "debug", key: "environment")
-            #else
-            scope.setTag(value: "prod", key: "environment")
-            #endif
-        }
+        if let sentry = sentryConfig {
+            SentrySDK.configureScope { scope in
+                #if DEBUG
+                scope.setTag(value: "debug", key: "environment")
+                #else
+                scope.setTag(value: "prod", key: "environment")
+                #endif
+            }
 
-        SentrySDK.start { options in
-            options.dsn = config.sentryKey
-            options.debug = config.sentryDebug
-            options.attachStacktrace = config.sentryAttachStacktrace
-            options.enableAutoSessionTracking = config.sentryAutoSessionTracking
-            options.tracesSampleRate = 0.0
-        }
+            SentrySDK.start { options in
+                options.dsn = sentry.key
+                options.debug = sentry.debug
+                options.attachStacktrace = sentry.attachStackTrace
+                options.enableAutoSessionTracking = sentry.autoSessionTracking
+                options.tracesSampleRate = 0.0
```

Comment:
Yeah, agreed with Jeremy that we should move this one into the Sentry config. I should've done this when I made this change last week.